README.md

```diff
@@ -88,6 +88,7 @@ itself) to look for plugins.
 | Jsonnet   | [Banno/asdf-jsonnet](https://github.com/Banno/asdf-jsonnet) | [![Build Status](https://travis-ci.org/Banno/asdf-jsonnet.svg?branch=master)](https://travis-ci.org/Banno/asdf-jsonnet)
 | Java      | [halcyon/asdf-java](https://github.com/halcyon/asdf-java) | [![Build Status](https://travis-ci.org/halcyon/asdf-java.svg?branch=master)](https://travis-ci.org/halcyon/asdf-java)
 | Julia     | [rkyleg/asdf-julia](https://github.com/rkyleg/asdf-julia) | [![Build Status](https://travis-ci.org/rkyleg/asdf-julia.svg?branch=master)](https://travis-ci.org/rkyleg/asdf-julia)
+| k6        | [grimoh/asdf-k6](https://github.com/grimoh/asdf-k6) | [![Build Status](https://travis-ci.org/grimoh/asdf-k6.svg?branch=master)](https://travis-ci.org/grimoh/asdf-k6)
```

Comment:
Could you replace CI badge to GitHub Actions one?

Reply:
Fixed.

Reply:
Thanks.